chart/templates/server-deployment.yaml

```diff
@@ -40,9 +40,33 @@ spec:
       securityContext:
         fsGroup: {{ $root.Values.fsGroup }}
       volumes:
+        - name: vets-api-mockdata
+          emptyDir: { }
+        - name: clamav-volume
+          emptyDir: { }
+        - name: ddsocket
+          hostPath:
+            path: /var/run/datadog
 {{ toYaml $root.Values.common.volumes | indent 8 }}
       {{- if $root.Values.mockData.enabled }}
       initContainers:
+        - name: vets-api-mockdata
+          image: 008577686731.dkr.ecr.us-gov-west-1.amazonaws.com/dsva/git:latest
```

Comment:
Image tag should not be public since this is hosted on a public gh page

Reply:
I really don't agree on this one - but removing this instance anyways.